charts/eoapi/values.yaml

```diff
@@ -492,6 +492,26 @@ previousVersion: ""
 ######################
 # NOTIFICATIONS
 ######################
-notifications:
+eoapi-notifier:
+  enabled: false
   sources:
     pgstac: false
+  config:
+    logLevel: INFO
+    sources:
+      - type: postgres
+        config:
+          host: ""  # Set dynamically via template
+          port: 5432
+          database: eoapi
+          username: eoapi
+          channel: pgstac_items_change
+    outputs:
```

Comment:
I would change this to something like:
```
    outputs:
      - type: cloudevents
        config:
          source: /eoapi/pgstac
          event_type: org.eoapi.stac.item
          destination:
             ref:
                 apiVersion: messaging.knative.dev/v1
                 kind: Broker
                 name: my-channel-1
                 namespace: serverless
```

Reply:
The ideea is that when you create in a `SinkBinding` in your helm chart it will result in a fully qualified URL to the Broker/Channel/Service, whatever else is referenced in the `ref` specification.

Also, desides `ref` I would also add `url` which should be mutually exclusive with ref. In case iti is set it should be an URL to which you can `POST` the CloudEvent.